__tests__/addressStorage.test.js

```diff
@@ -49,26 +49,4 @@ describe("address storage tests", function () {
       done();
     });
   });
-
-  it("will not switch address if address switching disabled", async () => {
```

Comment:
if the test should be passing but isn’t, then this should use it.skip instead

Reply:
turned out the test was not valid.